backend/services/llm/completion_service.py

```
@@ -228,9 +228,16 @@ async def execute_non_streaming_completion(
         # Build completion parameters
         completion_params = self.build_completion_params(request, api_key, api_base_url, stream=False)
         
-        # Debug logging
-        self.logger.info(f"Calling any-llm completion with params: {completion_params}")
-        self.logger.info(f"Provider: '{request.provider}', Model: '{request.model}', Model Identifier: '{completion_params['model']}'")
+        # Debug logging - DETAILED CONFIG BEFORE SENDING REQUEST
+        self.logger.info(f"=" * 80)
```

Ruff (0.14.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)